deployment/logging.md

```diff
@@ -148,13 +148,29 @@ end
 
 ## Output to Log File
 
+### By Command Line Option
+
 By default, Fluentd outputs to the standard output. Use `-o` command line option to specify the file instead:
 
 ```text
 $ fluentd -o /path/to/log_file
 ```
 
-### Log Rotation Setting
+### By Config File
+
+You can also configure the log file path using the `<log>` directive under `<system>`:
```

Comment:
How about adding the version info?

```suggestion
Since v1.18.0, You can also configure the log file path using the `<log>` directive under `<system>`:
```